draft-ietf-rats-corim.md

```diff
@@ -1280,14 +1280,14 @@ IANA is requested to allocate the following tags in the "CBOR Tags" registry {{!
 
 |     Tag | Data Item           | Semantics                                                            | Reference |
 |     --- | ---------           | ---------                                                            | --------- |
-|     500 | `tag`               | A corim, one out of $concise-rim-type-choice, see {{sec-corim-tags}} | {{&SELF}} |
-|     501 | `map`               | A corim-map, see {{sec-corim-map}}                                   | {{&SELF}} |
-|     502 | `tag`               | A signed-corim, see {{sec-corim-signed}}                             | {{&SELF}} |
+|     500 | `tag`               | A tagged-concise-rim-type-choice, see {{sec-corim-tags}} | {{&SELF}} |
```

Comment:
thanks for addressing my comment.  Please keep the existing typographic alignment though.